doc/.vale.ini

```diff
@@ -29,4 +29,6 @@ BasedOnStyles = Vale, Google
 
 # Removing Google-specific rule - Not applicable under some circumstances
 Google.WordList = NO
-Google.Colons = NO
\ No newline at end of file
+Google.Colons = NO
+Google.OxfordComma = NO
+Google.Headings = NO
```

Comment:
Yes! Sorry, I knew this. I should have remembered.

Reply:
No issues at all.